### PR TITLE
[RD-PRICING] update docs heston greeks

### PR DESCRIPTION
**Pull request by lacagninag**

### CODE REVIEW

File: Heston/Documentation/Heston.tex [hunk 1]
```diff
@@ -8,13 +8,14 @@
...
+This plug-in implements the Heston model. Once installed the plug-in offers the possibility of using two new processes, the Heston process and the Heston time dependent drift process. 
+The latter is a generalization of the former in which the drift is time-dependent. For a general reference on the Heston model see \cite{Heston:ClosedFormSol}.
 
 \section{How to use the plug-in}
 
 \subsection{Heston process}
-in the fairmat user interface, while adding a new stochastic process, Heston model can be used.
-The stochastic process is defined by the parameters shown in table below:
+in the fairmat user interface, while adding a new stochastic process, Heston model can be selected.
+The stochastic process is defined by the parameters shown in table the below:
```
Comment (andreaver) [inline Heston/Documentation/Heston.tex]: invertito l'ordine

File: Heston/Documentation/Heston.tex [hunk 2]
```diff
@@ -32,12 +33,12 @@ \subsection{Heston process}
...
 \end{center}
-$S_0$ and $V_0$ are the starting values for the stock process and the volatility process, the others are parameters regulating the model dynamic.
+$S_0$ and $V_0$ are the starting values for the stock process and the volatility process, the remaining parameters regulate the model dynamic.
 
 It is possible to edit the model values shown in the table above directly on the ``Parameters'' tab of the Heston process window by double clicking on them. To set the value of the correlation parameter (called $\rho$ in the following) you have to select the Heston process in the ``Stochastic Process'' window and then click on ``Correlation''.
 
 \subsection{Heston with time dependent drift}
-in the fairmat user interface, while adding a new stochastic process, Heston model with time dependent drift can be used.
+in the fairmat user interface, while adding a new stochastic process, Heston model with time dependent drift can be selected.
 The stochastic process is defined by the parameters shown in table below:
```
Comment (andreaver) [inline Heston/Documentation/Heston.tex]: in the table

File: Heston/Documentation/Heston.tex [hunk 3]
```diff
@@ -83,7 +84,7 @@ \section{Implementation Details}
 Parameters $k$, $\theta$ and $\sigma$ have to satisfy the constrain $2k\theta>\sigma^2$ (known as the Feller condition) in order to exclude the possibility for $V(t)$ to reach 0.
 
 In the case of Heston with constant drift $\mu$ is set equal to $r-q$ where $r$ is the risk free rate and $q$ is the dividend yield rate of the stock. 
-Please be careful to use a constant discount with risk free rate equal to the $r$ parameter.
+Please note that a constant discount factor should be selected while using a constant risk free rate.
```
Comment (andreaver) [inline Heston/Documentation/Heston.tex]: Please note that a constant discount factor should be selected while using the constant risk free rate $r$

**Comment by lacagninag** [comment]
sistemato

**Pull request merged**